src/math/integerHistogram1D.cpp

```diff
@@ -143,7 +143,7 @@ Data1D IntegerHistogram1D::data() const
     return result;
 }
 
-std::map<int, SampledDouble> IntegerHistogram1D::averages() const { return averages_; }
+std::map<int, SampledDouble> IntegerHistogram1D::averages() const & { return averages_; }
```

Comment:
Sorry, wasn't clear! I actually meant the return value should be `const &`:
```suggestion
// Return averages map
const std::map<int, SampledDouble> &IntegerHistogram1D::averages() const { return averages_; }
```